src/Metadata/Extractor/ResourceExtractorTrait.php

```diff
@@ -25,7 +25,7 @@ trait ResourceExtractorTrait
 {
     private function buildArrayValue(\SimpleXMLElement|array|null $resource, string $key, mixed $default = null): ?array
     {
-        if (\is_object($resource) && $resource instanceof \SimpleXMLElement) {
```

Comment:
Since the param is `\SimpleXMLElement|array|null`

is object will enfonce it's an SimpleXMLElement